src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/enable-logs-monitored-containers.mdx

```diff
@@ -0,0 +1,103 @@
+freshnessValidatedDate: never
+---
+
+import infrastructureAddNamesLogs from 'images/infrastructure_screenshot-crop_add-names-logs.webp'
+
+import infrastructureAddContainersLogPath from 'images/infrastructure_screenshot-crop_add-containers-log-path.webp'
+
+import logsParsingUi from 'images/infrastructure_screenshot-full_logs-parsing-ui.webp'
+
+Our [infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring) can gather logs from your containers. When you enable logs from the infrastructure agent, the agent will automatically assign those logs to the hosts running Docker.
+
+## Requirements [#requirements]
+
+To collect logs from your containers, you need to have:
+
+    * Infrastructure agent [1.8.32](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1832) or higher running on Linux
+    * CentOS version 6.0 or higher, if you're using CentOS
+    * Docker with engine from [v1.12](https://docs.docker.com/engine/release-notes/prior-releases/#1120-2016-07-28) or other [`containerd`-based container runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes)
+
+## Collect logs from your containers [#collect-logs]
+
+1. Open the configuration file for the infrastructure agent and add this snippet:
+
+    ```yml
+    - name: containers
+      file: /var/lib/docker/containers/*/*.log
+    ```
+
+<img
+    title="Add the containers path to your snippet"
+    alt="A screenshot that shows where to add this snippet in context of your yaml file"
+    src={infrastructureAddContainersLogPath}
+/>
+
+* You may find this file at this path: `/etc/newrelic-infra/logging.d/logging.yml`.
+* This section directs the agent to collect logs from your containers
+
+2. Add a tag section to your Docker configuration that passes in the default `{{.Name}}` attribute:
+
+* Docker run command example
+
+    ```shell
+    docker run --log-opt tag="{{.Name}}" -d log-generator
+    ```
+
+* Docker Compose example in `docker-compose.yml`
+
+    ```yml
+    version: '3.9'
+    x-default-logging: &logging
+      driver: "json-file"
+      options:
+        max-size: "5m"
+        max-file: "2"
+        tag: "{{.Name}}"
+    ```
+
+<img
+    title="Configuration file in context"
+    alt="Screenshot that shows this snippet in context of a configuration file"
+    src={infrastructureAddNamesLogs}
+/>
+
+This adds a `attrs.tag` attribute to your logs, which looks like this snippet:
+
+    ```json
+        {
+        agentName:Infrastructure
+        agentVersion:1.50.0
+        **attrs.tag:cart-service**
+        coreCount:1
+        ...
+        }
+    ```
+
+3. Once the `attrs.tag` appears in your host-level logs, you can enable container logs by going to <DoNotTranslate>**[one.newrelic.com](https://one.newrelic.com) > Infrastructure </DoNotTranslate>, then click <DoNotTranslate>**HERE**</DoNotTranslate>.
```

Comment:
What is `**HERE**`? Did you enable some way to somehow correlate the logs to a container instead of to a host?